### ISSUE TITLE
Last Events not updated

### ISSUE THREAD

**Issue by sjikkervisje** [matijse/eufy-ha-mqtt-bridge#90]
Just installed it following this video: https://youtu.be/x7JF2jgrvQE and just found out that it doesn't update. When Motion or person is detected it doesn't show the date/time and the image doesn't update. When restarting Home Assistant, only then the Image updates. Not even when I press the doorbell does the information update in Home Assistant. It seems that the Push is not coming through.

**Comment by davenbo** [comment]
I just installed via docker and can confirm what @sjikkervisje said. Having the same issue. No update in HA upon new events.

**Comment by sjikkervisje** [comment]
The battery percentage always shows 53% and doesn't even update to the actual value when I restart HA. The only thing it refresehs is the picture after a restart.

**Comment by davenbo** [comment]
> Found this one [https://github.com/fuatakgun/eufy_security](url) which works for the most part, perhaps something can be learned from how they did it here. I get more entities also.

Yes but doesn't support motion entities which is why I came here in the first place :(

**Comment by sjikkervisje** [comment]
> > Found this one [https://github.com/fuatakgun/eufy_security](url) which works for the most part, perhaps something can be learned from how they did it here. I get more entities also.
> 
> Yes but doesn't support motion entities which is why I came here in the first place :(

It actually does and it also works see below. The motion sensor depends on how you setup the motion detection. If set to all motion only the motion detected entitity trigggers, if set to person only then only the person detection motion sensor triggers. Unfortunately not all. Ps. I only have the Wireless Doorbell, no other cameras.

![image](https://user-images.githubusercontent.com/1422692/131311191-6da74f5a-53d2-4713-9d92-67819d8bc455.png)

**Comment by davenbo** [comment]
Ah I see. Thanks, tried it out and you are indeed correct. It's actually
all I need so dropping the mqtt bridge for now.

**Comment by MechEng70** [comment]
Seeing this issue with the Doorbell and Solo CAM S40.  The logs do show that "Someone has been spotted", however the image is not updated until the integration is restarted.

**Comment by MechEng70** [comment]
Looking at the logs, there is a push message that indicates that someone has been detected:

```
debug: Got notification - Device: T8124PXXXXXXXXXX, Type: 3102 {"timestamp":"2021-09-20T21:46:59.993Z"}
debug: Received push message {"pushMessage":{"id":"32D1D345","from":"348804314802","to":"eiBvb7KQnlbfLXY4uBLI9U","category":"com.oceanwing.battery.cam","persistentId":"0:1632174483617404%0d2a775cf9fd7ecd","ttl":3600,"sent":"1632174483608","payload":{"device_sn":"","payload":{"msg_type":18,"event_type":3102,"device_sn":"T8124P202131078E","name":"North - Carlson","channel":0,"cipher":28,"session_id":"20210920_144803","pic_url":"","create_time":1632174481212,"trigger_time":1632174481212,"file_path":"/media/mmcblk0p1/Camera00/20210920144803.h264","push_count":1,"notification_style":1,"storage_type":1,"unique_id":"40e751722c606c24bc09b726b9d3d8aa"},"station_sn":"T8124P202131078E","google.c.sender.id":"348804314802","type":"62","title":"eufyCam Solo","push_time":"1632174483585","content":"Someone has been spotted","event_time":"1632174483968"}},"timestamp":"2021-09-20T21:48:03.689Z"}
debug: Got notification - Device: T8124P202131078E, Type: 3102 {"timestamp":"2021-09-20T21:48:03.689Z"}
```

The image will not be updated until the Integration is restarted.  Is there a way to trigger the update?

**Comment by matijse** [comment]
There is no `pic_url` in your push message. In the app you can set different notification styles, did you choose the one with a picture? Do you see a picture in the app notifications? If so, then images may work differently for this camera...

**Comment by MechEng70** [comment]
> There is no `pic_url` in your push message. In the app you can set different notification styles, did you choose the one with a picture? Do you see a picture in the app notifications? If so, then images may work differently for this camera...

Just checked.  and you are correct.  The doorbell was set to "Full Effect: and the new cameras were set to "Most Efficient".  Resetting all to "Full Effect" and will monitor today.  Thanks.

**Comment by Rogiertw** [comment]
> 

Do you already have a solution for updating the actual battery status? I have the same issue, it never gets updated. It remains 99% in HA, while the Eufy app tells me it is 96%
